.github/workflows/mapi.yml

```diff
@@ -44,7 +44,20 @@ jobs:
 
     # Kill python if it's still running, ignoring any errors
     - name: Shut down API
-      run: pgrep python3 | xargs kill || true
+      run: pgrep coverage | xargs kill || true; sleep 5
```

Comment:
Why do we have a `sleep 5` here?

Reply:
The sleep 5 was because the python process is running in the background & wanted to make sure it closed before hitting the next next. sleep 1 would also probably be ok.

Reply:
(Coverage failed if process didn't get enough time to close)